packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-ciemss-drilldown.vue

```diff
@@ -392,24 +432,29 @@
 							@selection-change="updateChartSettings"
 						/>
 						<Divider />
-						<tera-chart-settings
-							:title="'Comparison charts'"
-							:settings="chartSettings"
-							:type="ChartSettingType.VARIABLE_COMPARISON"
-							:select-options="simulationChartOptions"
-							:selected-options="comparisonChartsSettingsSelection"
-							@open="setActiveChartSettings($event)"
-							@remove="removeChartSettings"
-							@selection-change="comparisonChartsSettingsSelection = $event"
-						/>
+						<!-- Comparison charts -->
 						<div>
+							<h5 class="pb-1">Comparison charts</h5>
+							<template v-for="setting in selectedComparisonChartSettings" :key="setting.id">
```

Comment:
Hm.. this feels a bit odd. Looping over settings should be inside <tera-chart-settings component.